src/Native/Runtime/unix/unixasmoffsets.cpp

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#ifdef _ARM_
+
+#define PLAT_ASM_OFFSET(offset, cls, member) OFFSETOF__##cls##__##member 0x##offset##h
```

Comment:
That's a result of me failing to get GAS assembly to treat symbols as immediate values and trying every permutation of specifying a hex literal :)  I'll change it to use just 0x (if that indeed works).